pyannote/audio/core/inference.py

```diff
@@ -387,7 +385,6 @@ def crop(
         """
 
         waveform, sample_rate = self.model.audio.crop(file, chunk, fixed=fixed)
-        # TODO remove this conversion if/when we switch to torchaudio IO
         waveform = torch.tensor(waveform, requires_grad=False)
```

Comment:
I think this is not needed since we wrap inference in a `with torch.no_grad:` in `Inference.infer` anyway.
That being said, calling `waveform.requires_grad_(False)` is probably better, isn't it?